### PR TITLE
0123 routine fix doc bug

### PR DESCRIPTION
**Pull request by haowang101779990**
# Routine doc bugs fix 

@dzhwinter Could you please help us review the changes in "metrics.rst" ? Thanks a lot !

@chengduoZH  I have fixed the bug that you have come up with this week. Please refer to "save_load_variables.rst" . Thank you very much!

### CODE REVIEW

**Review by chengduoZH — COMMENTED** [review]
LGTM

**Review by dzhwinter — APPROVED** [review]
LGTM

File: doc/fluid/user_guides/howto/evaluation_and_debugging/evaluation/metrics.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 模型评估
 ############
 
-模型评估是用指标反映模型在预期目标下精度，根据模型任务决定观察指标，作为在训练中调整超参数，评估模型效果的重要依据。
+模型评估是用指标(metrics)反映模型在预期目标下精度.其中，指标(metrics)根据模型任务决定。模型评估可作为在训练中调整超参数，评估模型效果的重要依据。
```
Comment (shanyi15) [inline doc/fluid/user_guides/howto/evaluation_and_debugging/evaluation/metrics.rst]: 请用中文句号

Comment (shanyi15) [reply doc/fluid/user_guides/howto/evaluation_and_debugging/evaluation/metrics.rst]: “模型评估可作为在训练中调整超参数”后面用顿号

File: doc/fluid/user_guides/howto/evaluation_and_debugging/evaluation/metrics.rst [hunk 2]
```diff
@@ -15,8 +15,8 @@ paddle.fluid.metrics模块包含该功能
 
 metric函数根据模型任务不同，指标构建方法因任务而异。
 
-回归类型任务labels是实数，因此loss和metric函数构建相同，可参考MSE的方法。
-分类任务常用指标为分类指标，本文提到的一般是二分类指标，多分类和多标签需要查看对应的API文档。例如排序指标auc，多分类可以作为0，1分类任务，auc指标仍然适用。
+回归类型任务labels是实数，可参考MSE的方法。
```
Comment (shanyi15) [inline doc/fluid/user_guides/howto/evaluation_and_debugging/evaluation/metrics.rst]: 这是本文里MSE第一次出现吗？如果是，请后面跟括号，写全称

**Review by shanyi15 — APPROVED** [review]
LGTM

**Comment by shanyi15** [comment]
please cherrypick to release/1.2 branch if necessary @haowang101779990

**Pull request merged**